determined-ai/determined
# test: playwright e2e test setup

## Description
[WEB-1113]

Added the following E2E test cases. 
- logging in
- navigation smoke test
- redirect to correct link after login

These are very minimal for now. we can extend it later.





## Test Plan
- Check if test cases are valid (didnt add edge cases since this is more like vital check)
- Check if CI works
- Check if E2E test works on local. [How to run test locally](https://github.com/determined-ai/determined/blob/292f3f703e65ab8d13e8d279d3de9d995fcd6eea/webui/react/src/e2e/README.md)
- Check if test is not flaky




## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket






[WEB-1113]: https://hpe-aiatscale.atlassian.net/browse/WEB-1113?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
